CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.0]
+### Added
+- Integrate `PollingController` mixin into the `SmartTransactionsController` and implement the abstract `_executePoll` method ([#210](https://github.com/MetaMask/smart-transactions-controller/pull/210))
+
```

Comment:
We might want to make this explicit for extra clarity. 

```suggestion
- **BREAKING**: Add `@metamask/polling-controller` ^0.2.0 as a dependency. 

```

Reply:
Actually realizing there is a breaking change in this PR. The controller now expects 'getNetworkClientById' passed from the NetworkController via the constructor...

Reply:
Yes. But it's quite late for me so will do in the morning

Reply:
done here: https://github.com/MetaMask/smart-transactions-controller/pull/216/commits/da94c3c950bd228cfd87e866d10ed2478a6ac144